utils/networks.go

```diff
@@ -10,13 +10,203 @@ type Network string
 // NetworkID represents the network ID as an unsigned 64-bit integer.
 type NetworkID uint64
 
-// Predefined network IDs.
+// Predefined network IDs and network strings.
 var (
 	// EthereumNetworkID is the ID for the Ethereum network.
 	EthereumNetworkID NetworkID = 1
-
-	// EthereumNetwork is the string representation of the Ethereum network.
 	EthereumNetwork Network = "ethereum"
+
+	// GoerliNetworkID is the ID for the Goerli network.
+	GoerliNetworkID NetworkID = 5
+	GoerliNetwork Network = "goerli"
+
+	// OptimismNetworkID is the ID for the Optimism network.
+	OptimismNetworkID NetworkID = 10
+	OptimismNetwork Network = "optimism"
+
+	// FlareNetworkID is the ID for the Flare network.
+	FlareNetworkID NetworkID = 14
+	FlareNetwork Network = "flare"
+
+	// RootstockNetworkID is the ID for the Rootstock network.
+	RootstockNetworkID NetworkID = 30
+	RootstockNetwork Network = "rootstock"
+
+	// LuksoNetworkID is the ID for the Lukso network.
+	LuksoNetworkID NetworkID = 42
+	LuksoNetwork Network = "lukso"
+
+	// CrabNetworkID is the ID for the Crab network.
+	CrabNetworkID NetworkID = 44
+	CrabNetwork Network = "crab"
+
+	// DarwiniaNetworkID is the ID for the Darwinia network.
+	DarwiniaNetworkID NetworkID = 46
+	DarwiniaNetwork Network = "darwinia"
```

Comment:
Note, these names don't convert to the string representation in the url, they're more just the human readable version.